Packs/CommonScripts/Scripts/IsInCidrRanges/IsInCidrRanges.py

```diff
@@ -1,19 +1,30 @@
-import demistomock as demisto
-from CommonServerPython import *
+import ipaddress
 
-from netaddr import IPAddress, IPNetwork
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
 
 
 def main():
-    ip_address = demisto.args()['left']
+    """ Check if given IP (or IPs) address is part of a given CIDR (or a list of CIDRs).
+
+    Args:
+        ip_addresses (str): A list of IPs/IPv6s
+        cidr_range_list (str): A list of CIDRs to be checked against.
+
+    Returns:
+        bool: True if given IP is part of given CIDR range.
+    """
+
+    ip_addresses = argToList(demisto.args()['left'])
```

Comment:
It is expected to be a list of IP addresses? If so, mention it in the yml and README files.

Reply:
Done